### PR TITLE
Sajiyah update

### PR DESCRIPTION
**Pull request by Sajiyah-Salat**
I am updating my old profile. 

 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4829"></a>

### CODE REVIEW

**Comment by Sajiyah-Salat** [comment]
I have tried and its failing again and agian. Let me know the reason. Thank you.

File: data/Sajiyah-Salat.json [hunk 1]
```diff
@@ -1,34 +1,106 @@
...
+      "color": "blue",
+      "description": "I post daily about my learnings on twitter and linkedin. Many poeple support my work and I have made great connections."
+      "url": "https://www.linkedin.com/in/sajiya-salat-0a2a78245/",
+    },
+      {
+      "title": "Started Writitng Blogs",
+      "date": "October 2022",
+      "icon": "FaPencilAlt",
+      "color": "blue",
+      "description": "My blog are my stories of how I achieved a certain height. Many poeple got inspired by my journey."
```
Comment (swagataroy30) [inline data/Sajiyah-Salat.json:78]:
```suggestion
      "description": "My blog are my stories of how I achieved a certain height. Many poeple got inspired by my journey.",
```

File: data/Sajiyah-Salat.json [hunk 2]
```diff
@@ -1,34 +1,106 @@
...
+      "url": "https://medium.com/@Sajiyah-Salat"
+    },
+    {
+      "title": "Made Three DevOps Project",
+      "date": "November 2022",
+      "icon": "SiGithub",
+      "color": "black",
+      "description": "After Completing my DevOps Basics. I started making Projects. My projects mainly contain the knowledge of Docker, Kubernetes, AWS, Golang. My Projects are live on Github."
```
Comment (swagataroy30) [inline data/Sajiyah-Salat.json:86]:
```suggestion
      "description": "After Completing my DevOps Basics. I started making Projects. My projects mainly contain the knowledge of Docker, Kubernetes, AWS, Golang. My Projects are live on Github.",
```

File: data/Sajiyah-Salat.json [hunk 3]
```diff
@@ -1,34 +1,106 @@
...
+      "url": "https://github.com/Sajiyah-Salat",
+    },
+    {
+      "title": "Completed cka course from kodekloud",
+      "date": "December 2022",
+      "icon": "FaPencilAlt",
+      "color": "blue",
+      "description": "DevOps is my passion and I have completd all the basics about it. I have a good hold in kuberenetes and soon I will give cka(certified kubernetes administrator) exam. I have written a blogs on this topic."
```
Comment (swagataroy30) [inline data/Sajiyah-Salat.json:94]:
```suggestion
      "description": "DevOps is my passion and I have completd all the basics about it. I have a good hold in kuberenetes and soon I will give cka(certified kubernetes administrator) exam. I have written a blogs on this topic.",
```

File: data/Sajiyah-Salat.json [hunk 4]
```diff
@@ -1,34 +1,106 @@
...
+      "url": "https://medium.com/@Sajiyah-Salat"
+    },
+    {
+      "title": "Started my youtube video",
+      "date": "February 2022",
+      "icon": "SiYoutube",
+      "color": "red",
+      "description": "This youtube channel will be based on github contributions and community work. If you are just starting out in open source, I have many things to serve you."
```
Comment (swagataroy30) [inline data/Sajiyah-Salat.json:102]:
```suggestion
      "description": "This youtube channel will be based on github contributions and community work. If you are just starting out in open source, I have many things to serve you.",
```

**Review by swagataroy30 — COMMENTED** [review]
You need to add comma at line 70, 78, 86, 94, 102

File: data/Sajiyah-Salat.json [hunk 5]
```diff
@@ -1,34 +1,106 @@
...
+      "description": "Made Github profile on June 2022 | Started Contributing in Multiple Projects.",
+      "url": "https://github.com/Sajiyah-Salat"
+    },
+      {
+      "title": "Started Documenting my journey on twitter and linkedin",
+      "date": "June 2022",
+      "icon": "SiLinkedin",
+      "color": "blue",
+      "description": "I post daily about my learnings on twitter and linkedin. Many poeple support my work and I have made great connections."
+      "url": "https://www.linkedin.com/in/sajiya-salat-0a2a78245/",
```
Comment (swagataroy30) [inline data/Sajiyah-Salat.json:71]:
```suggestion
      "url": "https://www.linkedin.com/in/sajiya-salat-0a2a78245/"
```

File: data/Sajiyah-Salat.json [hunk 6]
```diff
@@ -1,34 +1,106 @@
...
+      "description": "My blog are my stories of how I achieved a certain height. Many poeple got inspired by my journey."
+      "url": "https://medium.com/@Sajiyah-Salat"
+    },
+    {
+      "title": "Made Three DevOps Project",
+      "date": "November 2022",
+      "icon": "SiGithub",
+      "color": "black",
+      "description": "After Completing my DevOps Basics. I started making Projects. My projects mainly contain the knowledge of Docker, Kubernetes, AWS, Golang. My Projects are live on Github."
+      "url": "https://github.com/Sajiyah-Salat",
```
Comment (swagataroy30) [inline data/Sajiyah-Salat.json:87]:
```suggestion
      "url": "https://github.com/Sajiyah-Salat"
```

**Review by swagataroy30 — COMMENTED** [review]
You need to remove the comma from line 71 and 87. Thank You!

**Review by swagataroy30 — COMMENTED** [review]
You need to add comma at line 70, 78, 86, 94, 102. Thank You!

File: data/Sajiyah-Salat.json [hunk 7]
```diff
@@ -1,34 +1,106 @@
...
+      {
+      "title": "Started Documenting my journey on twitter and linkedin",
+      "date": "June 2022",
+      "icon": "SiLinkedin",
+      "color": "blue",
+      "description": "I post daily about my learnings on twitter and linkedin. Many poeple support my work and I have made great connections.",
+      "url": "https://www.linkedin.com/in/sajiya-salat-0a2a78245/"
+    },
+      {
+      "title": "Started Writitng Blogs",
```
Comment (Ananya2001-an) [inline data/Sajiyah-Salat.json:74]: you can fix this typo in title
```suggestion
      "title": "Started Writing Blogs",
```

**Review by Ananya2001-an — APPROVED** [review]
Looks good.

**Comment by Sajiyah-Salat** [comment]
I hope it get merged soon.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**